Modules/Scripted/DICOMLib/Resources/DICOM/dcmtk/storescu.cfg

```diff
@@ -0,0 +1,62 @@
+#
+#  Copyright (C) 2003-2021, 3DSlicer
+#
+#  Author:  Rob Oakes
+#
+#  Purpose: Storescu configuration optimized for DICOM-SR and DICOM-SEG files
+#
+
+# ============================================================================
+[[TransferSyntaxes]]
+# ============================================================================
+
+[Uncompressed]
+TransferSyntax1 = LittleEndianExplicit
+TransferSyntax2 = BigEndianExplicit
+TransferSyntax3 = LittleEndianImplicit
+
+[JPEGBaseline]
+TransferSyntax1 = JPEGBaseline
+
+[JPEGLossless]
+TransferSyntax1 = JPEGLossless:Non-hierarchical-1stOrderPrediction
+
+[RLE]
+TransferSyntax1 = RLELossless
+
+[MPEG2]
+TransferSyntax1 = MPEG2MainProfile@MainLevel
+#
+# commented out the following line since we do not support transcoding of MPEG2
+#
+#TransferSyntax2 = MPEG2MainProfile@HighLevel
+
+[MPEG4]
+TransferSyntax1 = MPEG4HighProfile/Level4.1
+#
+# commented out the following lines since we do not support transcoding of MPEG4
+#
+#TransferSyntax2 = MPEG4BDcompatibleHighProfile/Level4.1
+#TransferSyntax3 = MPEG4HighProfile/Level4.2For2DVideo
+#TransferSyntax4 = MPEG4HighProfile/Level4.2For3DVideo
+#TransferSyntax5 = MPEG4StereoHighProfile/Level4.2
+
+# ============================================================================
+[[PresentationContexts]]
+# ============================================================================
+
+[StorageCompressedAndUncompressed]
+#
+# For DICOM-SR and DICOM-SEg SOP classes, only uncompressed transmission is supported.
+#
+#
+# Segmentation transfer syntaxes
+#
+PresentationContext1 = 1.2.840.10008.5.1.4.1.1.66.4\Uncompressed
+
+# ============================================================================
+[[Profiles]]
```

Comment:
Add a link to the original, unmodified DCMTK configuration and mark very clearly which lines were added/modified compared to the original and why. This is important because time to time we will have to update the configuration from DCMTK and re-apply all the modifications manually.

Reply:
@lassoan  Thank you for the comments and feedback. I'll incorporate your suggestions and push a new branch tomorrow.

Reply:
It would be great if we could finalize this pull request while our memory is still fresh and there are no merge conflicts.

Reply:
> Add a link to the original, unmodified DCMTK configuration and mark very clearly which lines were added/modified compared to the original and why. This is important because time to time we will have to update the configuration from DCMTK and re-apply all the modifications manually.

The configuration file, `storescu.cfg`, provides the set of DICOM transfer options that are sent as part of the DICOM presentation context. This includes what types of rules and compression options are supported and the storage modalities that are presented to the PACS.

By far the most important component of the file is the list of presentation contexts found under the `PresentationContexts.StorageCompressedAndUncompressed`. I'm not sure if it's part of the DICOM standard or for some other reason, but storescu chose to hard code 128 values covering some of the most common storage modalities.

To make the alternative configuration I provided, I grabbed a copy of the DCMTK config that was included as part of the development source, removed nearly all of the presentation contexts and added the modalities for the DICOM seg files I needed. My thinking was that as additional modalities are discovered that are part of 3DSlicer's core feature set they can just be added by creating additional context values. Examples:

```ini
...
PresentationContext2 = SegmentationStorage
PresentationContext3 = SegmentedVolumeRenderingVolumetricPresentationStateStorage
PresentationContext4 = SurfaceScanPointCloudStorage
PresentationContext5 = SurfaceSegmentationStorage
...
```

My question here: because it is not controlled by Slicer, is it better to include a copy of the original config in our sources, or to clearly document where it came from and the extensions needed?

Reply:
> It would be great if we could finalize this pull request while our memory is still fresh and there are no merge conflicts.

Sorry for the delay, I ended up getting pulled away because of some needs on a separate project. I'm hoping to get back to this cleaned up tomorrow or Tuesday.

Reply:
> It would be great if we could finalize this pull request while our memory is still fresh and there are no merge conflicts.

I've pulled the most recent changes from the `master` branch, which hopefully should resolve any potential merge conflicts. Please let me know if you would like me to restructure the PR.

Reply:
Everything should be ready for a second review.